README.md

```diff
@@ -71,17 +71,19 @@ sqlx = { version = "0.2", default-features = false, features = [ "runtime-tokio"
 #### Cargo Feature Flags
 
  * `runtime-async-std` (on by default): Use the `async-std` runtime.
- 
+
  * `runtime-tokio`: Use the `tokio` runtime. Mutually exclusive with the `runtime-async-std` feature.
- 
+
  * `postgres`: Add support for the Postgres database server.
- 
+
  * `mysql`: Add support for the MySQL (and MariaDB) database server.
- 
+
  * `uuid`: Add support for UUID (in Postgres).
- 
+
  * `chrono`: Add support for date and time types from `chrono`.
- 
+
+ * `time`: Add support for date and time types from `time` crate (simpler than `chrono`)
```

Comment:
"simpler" is a little subjective, I think. I would rather say "a competitor/alternative (take your pick) to `chrono`".

We should also mention here that if both the `time` and `chrono` features are activated, the `query!()` macro will prefer types from `time`.

Reply:
done, but not quite sure about exact wording